specification/cognitiveservices/data-plane/Language/preview/2022-10-01-preview/common.json

```diff
@@ -0,0 +1,1634 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Microsoft Cognitive Language Service",
+    "description": "The language service API is a suite of natural language processing (NLP) skills built with best-in-class Microsoft machine learning algorithms.  The API can be used to analyze unstructured text for tasks such as sentiment analysis, key phrase extraction, language detection and question answering. Further documentation can be found in <a href=\"https://docs.microsoft.com/en-us/azure/cognitive-services/text-analytics/overview\">https://docs.microsoft.com/en-us/azure/cognitive-services/text-analytics/overview</a>.",
+    "version": "2022-10-01-preview"
+  },
+  "paths": {},
+  "definitions": {
+    "ErrorResponse": {
```

Comment:
>ErrorResponse [](http://example.com/codeflow?start=5&length=13)

Do you need to define this? Can't you just use the one from 
Instead of redeclaring, I'd instead reference the one from common-types/v3/types.json#Definitions/ErrorResponse
?

Reply:
Ah, just realized this is a data plane api review. I wonder if that is still really a good idea.

Reply:
We didn't know about common-types folder before. Good suggestion. We can do refactoring in the future.

Reply:
Generally, do not use control plane types in data plane. For swagger, what you're doing now is what we recommend. It will be different for Cadl but that's a separate topic.